js/plugins/trimTimeline.js

```diff
@@ -133,18 +134,24 @@ trimClinicalTimeline.prototype.run = function (timeline, spec) {
         .attr("class","kink-bg")
         .attr("x", kinkPosition - 17.5)
         .attr("y", 15)
-        .attr("fill", "white");
+        .attr("fill", "white")
 
       kink.append("path")
         .attr("d", kinkLine(kinkLineData))
         .attr("stroke", "black")
         .attr("stroke-width", 1)
         .attr("fill", "none")
         .attr("class","kink-line")
-        .attr("transform", "translate("+ (kinkPosition - 92.5) +",20)");
+        .attr("transform", "translate("+ (kinkPosition - 92.5) +",20)")
     }
   }
 
+  d3.select(".kink-bg, .kink-line")
```

Comment:
This selector would fail if you have two timelines with the kink enabled on the same page